src/PerfCounters.cc

```diff
@@ -100,6 +100,7 @@ enum CpuMicroarch {
   AMDZen2,
   AMDZen3,
   AMDZen4,
+  AMDRyzen9,
```

Comment:
Don't need to add this; this CPU is Zen 4.

Reply:
Try it. Not supported.

Reply:
Why did you put 0x6f there? That's the `llsc_cntr_event` and it makes no sense to set that for an x86 CPU.

Using Zen4 should work. If it doesn't, something else is quite broken.

Reply:
I've just followed the pattern I saw before. After compiling this version it works on my end, so maybe the issue is not fix by my did but was already fixed between the latest package release and the git version?

Reply:
Yeah, this change doesn't actually do anything because nothing detects this new architecture type AMDRyzen9..